_includes/manuals/1.18/1.2_release_notes.md

```diff
@@ -286,6 +286,8 @@ The following deprecations with target version 1.20 were added:
 
 ### Upgrade warnings
 
+The `trusted_puppetmaster_hosts` setting has been renamed to `trusted_hosts`.
+
 The following plugins are known to be incompatible with 1.18, their updated versions have not been released yet:
 
 * foreman_chef 0.7.0 and lower
```

Comment:
This can removed as well.

Reply:
ok, but i'd rather do that in a separate pr.

Reply:
Also, are you sure? foreman_chef 0.7 seems to already include some fixes for 1.17 compat, but i'm not sure about 1.18 (cc: @ares)

Reply:
This wasn't actually true at the release. Perhaps during RC1 we didn't have 0.7.0 but somewhere during the development cycle we didn't have foreman_chef packages that were compatible with the current code. It was fixed in https://github.com/theforeman/foreman-packaging/pull/2610